external/winrt/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
+      netstandard1.0;
+      netstandard1.2;
```

Comment:
Since the ns1.0 and ns1.2 configs of this project behave the same you only need 1.0.